src/ASM/SAMpatchPETSc.C

```diff
@@ -0,0 +1,213 @@
+// $Id$
+//==============================================================================
+//!
+//! \file SAMpatchPETSc.C
+//!
+//! \date Sep 6 2010
+//!
+//! \author Runar Holdahl / SINTEF
+//!
```

Comment:
Skal du la Runar få æren for denne nye klassen også? Syns vel det er på høy tid du tar crediten selv ifm. omleggingen.